assets/ca_configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    # This label ensures that the OpenShift Certificate Authority bundle
+    # is added to the ConfigMap.
+    config.openshift.io/inject-trusted-cabundle: "true"
+  name: vsphere-csi-driver-trusted-ca-bundle
+  namespace: openshift-cluster-csi-drivers
```

Comment:
Who is going to apply this ConfigMap?

Reply:
I think some cluster component populates this configmap. I don't know which one though.

Reply:
The network operator populates this ConfigMap once it exists in the cluster, but first something must create it, right? I don't see this operator creating/applying it.

Reply:
Good catch! Added to `WithStaticResourcesController()`